doc/source/commands/communication.rst

```diff
@@ -6,6 +6,22 @@ Communication
 kOS allows you to write scripts that communicate with scripts running on other processors within the same vessel
 (inter-processor communication) or on other vessels (inter-vessel communication).
 
+Limitations
+-----------
+
+First off, we have to discuss certain limitations of how communications in kOS can be conducted. Those limitations
+stem directly from how KSP is built and how it simulates the physical world.
+
+Whenever you switch to a vessel KSP simulates the vessel itself and anything that is less than 2.5 kilometers from it
+(including other vessel if there are any). Things that are outside of that range are not loaded into the simulation.
+KSP uses a simpler simulation method for them that only calculates their position and other most basic properties.
+KSP calls that "being on rails".
+
+This means kOS can't run two processors that are separated by more than 2.5km from each other at the same time.
+As a consequence it's impossible for them to communicate live. If you want to exchange messages between them
+you will have to switch from one vessel to the other. This can be troublesome if you plan on sending more
+than a few messages back and forth. Of course everything will work as expected when vessels are closer than 2.5km.
```

Comment:
It's often a challenge to make an explanation simultaneously simple and thruthful.  There's a great quote from Einstein (or at least often attributed to Einstein - maybe it's one of those fake quotes), that goes, "Everything should be made as simple as possible, but no simpler".  That's one of the problems I have with DasValdez's streams is that he doesn't mind making statements "simpler than possible" so they end up being misleading, and if anyone in the chat mentions an important qualifier that covers a case where what he said isn't really true, he chastises them for "confusing" the newbies with, you know, facts.

Anyway, what might be a good way to keep the explanation simple here without being false would be to just say something like "This is a greatly simplified explanation of what's really going on that's leaving out a few important qualifiers.  If you want to fully understand how this "on rails" system works, check the "loaddistance" documentation..."  With a link to the documentation page about loaddistance, which has the full description of the rails system and now messy it is.